internal/partitions/partition_test.go

```diff
@@ -23,8 +23,8 @@ func (suite *UnitTestSuite) TestPartitionFindCmd() {
 				{"afterClusterTime", startAt},
 			}},
 			{"noCursorTimeout", true},
-			{"filter", expectedFilter},
 			{"hint", bson.D{{"_id", 1}}},
+			{"filter", expectedFilter},
```

Comment:
I reordered the find options in the `GetFindOptions` method for code quality reason, so that the `filter` field is appended at the end for both cases instead of separately in each block of an `if` statement.